gradle.properties

```diff
@@ -31,10 +31,14 @@ gradleLombokPluginVersion=3.2.0
 gradleMkdocsPluginVersion=1.0.0
 gradleSshPluginVersion=2.9.0
 grpcVersion=1.17.1
-guavaVersion=28.1-jre
-hadoopVersion=2.8.5
+
+# Downgraded the Guava version, to avoid version conflicts with transitiv dependencies of HDFS. Guava versions above
+# this one do not have `com.google.common.util.concurrent.Futures` `addCallback(ListenableFuture, FutureCallback)`
+# method, which is in turn used by HDFS server class `org.apache.hadoop.hdfs.server.datanode.checker.ThrottledAsyncChecker`.
+guavaVersion=25.1-jre
```

Comment:
This is not acceptable. We should never downgrade dependencies because we could be reintroducing bugs that were solved by upgrading in the first place. 

Please find another solution to this problem.

Reply:
Yes, that makes sense. I've revert the change.